test/streamingReceiver.spec.ts

```diff
@@ -184,6 +188,9 @@ describe("Streaming Receiver Misc Tests", function(): void {
     }
 
     await receiveListener.stop();
+
+    chai.assert.fail(unexpectedError && unexpectedError.message);
```

Comment:
@ramya-rao-a 
This check(`chai.assert.fail`) is throwing the error even if the test passes(`unexpectedError = undefined`).
> AutoComplete removes the message from Partitioned Queues(with sessions):
     `AssertionError: assert.fail()`

Reply:
@HarshaNalluru That's right. This got fixed in https://github.com/Azure/azure-service-bus-node/commit/9e3c868557f17d1f751e403ee3553cf129ec63bb

But looks like there are 2 other files with this. I'll send a PR for that

Reply:
@HarshaNalluru Sent PR #217. If you take a look and approve, then I'll merge it